src/views/UploadFileView/Components/DragAndDrop.jsx

```diff
@@ -0,0 +1,93 @@
+import React from 'react'
+import { Button, Paper, Typography, makeStyles } from '@material-ui/core'
+import { useDropzone } from 'react-dropzone'
+import FolderIcon from '../../../components/Icons/FolderIcon'
+import { FileUpload } from '@mui/icons-material'
+import { uploadFileData } from '../../../service/uploadFileData'
+
+const useStyles = makeStyles({
+  root: {
+    textAlign: 'center',
+    padding: 20
+  },
+  container: {
+    borderStyle: 'dotted',
+    padding: 10,
+    borderColor: '#A38C8C'
+  },
+  title: {
+    fontSize: 16,
+    color: '#2f5487',
+    fontWeight: 'bold',
+    letterSpacing: '0.5px'
+  },
+  buttonContainer: {
+    textAlign: 'center'
+  },
+  fileInfo: {
+    fontSize: 16,
+    letterSpacing: '0.5px'
+  }
+})
+
+export default function DragAndDrop (props) {
+  const classes = useStyles()
+  const { acceptedFiles, getRootProps, getInputProps } =
+    useDropzone({
+      maxFiles: 1,
+      accept: '.csv,.xlsx'
+    })
+
+  const acceptedFileItems = acceptedFiles.map((file) => (
+    <li key={file.path}>
+      {file.path} - {file.size} bytes
+    </li>
+  ))
+
+  function getBinaryFromFile (file) {
+    return new Promise((resolve, reject) => {
+      const reader = new FileReader()
+      reader.addEventListener('load', () => resolve(reader.result))
+      reader.addEventListener('error', err => reject(err))
+      reader.readAsBinaryString(file)
+    })
+  }
+
+  const onClick = async (_) => {
+    const fileToUpload = acceptedFiles[0]
+    await uploadFileData({
```

Comment:
We receive a response from the backend, I think it is important to show the user a success or error message and delete the info from the file once a response is obtained.

Reply:
I added a snackbar that receives the response from the bakcend and depending on it displays a success or failure message. I also show a message when you try to upload files without having selected them.